pipeline-steps/deploy-service.yaml

```diff
@@ -22,9 +23,11 @@ steps:
       terraformInitSubscription: ${{ parameters.tfInitSub }}
       product: ${{ parameters.product }}
       tfVarsFile: "$(System.DefaultWorkingDirectory)/$(buildRepoSuffix)/environments/${{ parameters.stack }}/${{ parameters.environment }}.tfvars"
-      initCommandOptions: "-var subscription_id=$(ARM_SUBSCRIPTION_ID) -backend-config resource_group_name=$(backendAzureRmResourceGroupName)"
-      planCommandOptions: "-var control_vault=$(controlKeyVault) -var project=$(project) ${{ parameters.targetCommand }}"
-
+      initCommandOptions: >
+        -var subscription_id=$(terraformSubscriptionID)
+        -backend-config resource_group_name=$(backendAzureRmResourceGroupName)
+      planCommandOptions: "-var subscription_id=$(terraformSubscriptionID) -var control_vault=$(controlKeyVault) -var project=$(project) ${{ parameters.targetCommand }}"
```

Comment:
This format is needed, as with > it creates a blank space when attempting to use `${{ parameters.targetCommand }} `and it has no value